src/snipe/api/multisig_reference_QC.py

```
@@ -790,152 +770,137 @@ def sort_chromosomes(chrom_dict):
         if predict_extra_folds and genome_stats["Genome coverage index"] > 0.01:
             predicted_fold_coverage = {}
             predicted_fold_delta_coverage = {}
+            predicted_unique_hashes = {}
+            predicted_delta_unique_hashes = {}
```

Ruff

Local variable `predicted_delta_unique_hashes` is assigned to but never used

Remove assignment to unused variable `predicted_delta_unique_hashes`

(F841)